### ISSUE TITLE
Scrolling on Tavern page shows me the source code for the page

### ISSUE THREAD

**Issue by diyadas** [HabitRPG/habitica#5053]
I'm on Chrome for Mac (41.0.2272.118), and I think this may be a browser issue, since I've had this problem when viewing discussions on Apple's website. It's never happened with HabitRPG before though. Refreshing the page sometimes fixes the issue, but not always, and sometimes refreshing the (normally displayed) page gets me the source code. I tried Googling a fix, but I think I might be using the wrong terms, because the search results don't help.

User ID: 13018811-220f-4025-93a0-9ed6b6a21032

**Comment by deilann** [comment]
This is the kind of thing where a screenshot would be mighty helpful.

**Comment by diyadas** [comment]
Here you are:
![source-code](https://cloud.githubusercontent.com/assets/7118569/7238671/e60d45d2-e758-11e4-96ca-b23d36a9fe23.png)

**Comment by TheHollidayInn** [comment]
To clarify, when you visit the page you can see the normal view, but when you start to scroll down, you see the code? 

Also, can you give a post a screenshot that shows the whole browser window (if you don't mind)?

And finally, do you have any add-ons installed?

**Comment by diyadas** [comment]
It has magically fixed itself today, but to answer your questions:

It was actually specific to when I scrolled up - scrolling down was fine.
I couldn't view the entire page at once, but I managed to copy over the entire text yesterday - hopefully (this)[https://gist.githubusercontent.com/InfH/6c459e08e3e48763340c/raw/cf6e82d4785e80e83bcda86f11761aecd816a77c/gistfile1.txt] works.
I have lots of extensions enabled and I should have tried disabling them when I had the issue but I didn't think of that.
